docs/modules/ROOT/pages/visual-tour.adoc

```diff
@@ -332,14 +317,16 @@ For more information about keyboard shortcuts, use the command `:help keys`.
 [[reusable-frames]]
 == Reusable result frames
 
-Neo4j Browser reusable result frames allow you to edit the query of an existing result directly in the result frame and re-run to update the result in situ.
+Neo4j Browser reusable result frames allow you to edit the query of an existing result directly in the result frame and re-run it to update the result in situ.
 
 image:inline-result-editor.gif[]
 
-Each reusable frame maintains its own local history of commands and updates the main one in the Cypher editor, should you need to instantiate a new result frame from there.
+You can also use _Cmd + click_ to send it back to the main editor and re-run it from there.
```

Comment:
Worth documenting the non-mac use case too. Will be ctrl-click for Windows / Linux users.